Directory.Build.props

```diff
@@ -19,6 +19,7 @@
       https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
       https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
       https://dotnet.myget.org/F/roslyn-analyzers/api/v3/index.json;
+      https://pkgs.dev.azure.com/dnceng/public/_packaging/MachineLearning/nuget/v3/index.json;
```

Comment:
This should be removed.

Reply:
This goes along with question about project/package reference. I added this after the email we had yesterday, but it sounds like this should only exist in the feature branch and the code that gets checked into master wont have this. Is that a correct understanding?

---
In reply to: [340734033](https://github.com/dotnet/machinelearning/pull/4413#discussion_r340734033) [](ancestors = 340734033)

Reply:
Per the comment to my other question, I will revert this here as well and only have it in the feature branch.

---
In reply to: [340748790](https://github.com/dotnet/machinelearning/pull/4413#discussion_r340748790) [](ancestors = 340748790,340734033)